File: examples/FaxLineDeleteExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
)

with ApiClient(configuration) as api_client:
    fax_line_delete_request = models.FaxLineDeleteRequest(
        number="[FAX_NUMBER]",
    )

    try:
        api.FaxLineApi(api_client).fax_line_delete(
            fax_line_delete_request=fax_line_delete_request,
        )
    except ApiException as e:
        print("Exception when calling FaxLineApi#fax_line_delete: %s\n" % e)

File: examples/FaxLineDeleteExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class FaxLineDeleteExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";

        var faxLineDeleteRequest = new FaxLineDeleteRequest(
            number: "[FAX_NUMBER]"
        );

        try
        {
            new FaxLineApi(config).FaxLineDelete(
                faxLineDeleteRequest: faxLineDeleteRequest
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FaxLineApi#FaxLineDelete: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
